### PR TITLE
[NPT-9216] Return dict consistent with docs from get_logs()

### PR DESCRIPTION
**Pull request by aniezurawski**
Closes NPT-9216

### CODE REVIEW

File: alpha_integration_dev/old_client.py [hunk 1]
```diff
@@ -106,8 +106,11 @@ def log_series(self):
         neptune.log_image('g_img', self.img_path)
 
         # see what we've logged
+        from time import sleep
+        sleep(5)
```
Comment (shnela) [inline alpha_integration_dev/old_client.py:110]: Is there any equivalent of `exp.sync()` in old client?

File: alpha_integration_dev/old_client.py [hunk 2]
```diff
@@ -106,8 +106,11 @@ def log_series(self):
...
         logs = neptune.get_experiment().get_logs()
         print(f'Logs: {logs}')
+        assert logs == {'m1': '2.0', 'g_img': None, 'nested/m1': '1.0', 'm2': 'c'}
```
Comment (shnela) [inline alpha_integration_dev/old_client.py:113]: * Old API returns strings while new one integer/float values.
* No idea why `g_img` is `None`, but we have the same behavior for old client. `x` and `lastX` values for returned `g_img` channel is equal 1, so images has been uploaded.

File: neptune/internal/utils/alpha_integration.py [hunk 3]
```diff
@@ -140,12 +140,11 @@ def channelType(self):
             return ChannelType.IMAGE.value
 
     @property
-    def lastX(self):
-        # TODO: NPT-9216
-        return None
+    def x(self):
+        return self._properties.lastStep
```
Comment (shnela) [inline neptune/internal/utils/alpha_integration.py:144]: Here's no such value: `_properties.lastStep`.
None of `*SeriesAttributeDTO` contains it when returned from API.

**Review by shnela — CHANGES_REQUESTED** [review]
Can you start PR title with `[NPT-9216]`, please?

File: alpha_integration_dev/old_client.py (hunk 1 above)
Comment (aniezurawski) [reply alpha_integration_dev/old_client.py:110]: Nope.

File: neptune/internal/utils/alpha_integration.py (hunk 3 above)
Comment (aniezurawski) [reply neptune/internal/utils/alpha_integration.py:144]: Fetch and look again. ;)

File: alpha_integration_dev/old_client.py (hunk 2 above)
Comment (aniezurawski) [reply alpha_integration_dev/old_client.py:113]: 1. Fixed
2. We do not return images in old client by design. It was always this way.

File: neptune/internal/utils/alpha_integration.py (hunk 3 above)
Comment (shnela) [reply neptune/internal/utils/alpha_integration.py:144]: Hmmm, still no `x`  value on `stage` (screen).
image

But I assume it'll work after deploying `[e6f65a16] Add last step to series attributes ` from backend repo.

**Review by shnela — APPROVED** [review]
Assuming `[e6f65a16] Add last step to series attributes ` will solve the problem with channel `x` attribute, it's ok.

**Pull request merged**